src/Components/signUpPage.scss

```diff
@@ -0,0 +1,189 @@
+@import "../styles/main.scss";
+.signUp {
+  height: 700px;
+  width: 100%;
+  background-color: $black;
+  @include flex(column, center, center);
+  &__form {
+    @include flex(column, center, center);
+    width: 400px;
+    height: 500px;
+  }
+  &__wrapper {
+    @include flex(column, space-around, center);
+    width: 450px;
+    height: 100%;
+    background-color: $background-color;
+    border: 1px solid $border-color;
+    border-radius: 5px;
+    padding: 3rem;
+  }
+  &__wrapper:hover {
+    border: 1px solid $primary-color;
+    box-shadow: $primary-color 0px 5px 15px;
+  }
+
+  &__wrapper h1 {
+    text-align: left;
+    font-family: "Netflix Sans", sans-serif;
```

Comment:
dont use font family like this. Apply the font to the whole body. We should not have two different fonts. So if you apply it to the whole body then you dont need to do this manually. So you should remove all these and only apply it to the body in layout.scss